client/src/main/resources/templates/js/main.js

```diff
@@ -545,7 +545,7 @@ function displaySource(src, title) {
     var pageLoaded = 0;
     var i = startNumber;
     while ((i <= mNoOfPage) && (pageLoaded < 5)) {
-      $(paginationId + "testcase-pagination ul .page-next").before($('<li>').attr('data-index',i).addClass('page-number').append($('<a>').attr('href','#').append(i) ));
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1307322591/issues/source?bid=2928425&fileBranchId=2951451#l548)